examples/with-persistent/mu-example-with-persistent.cabal

```diff
@@ -0,0 +1,30 @@
+name:                mu-example-with-persistent
+version:             0.1.0.0
+-- synopsis:
+-- description:
+homepage:            https://github.com/higherkindness/mu-haskell/examples/with-persistent#readme
+author:              Flavio Corpa
+maintainer:          flavio.corpa@47deg.com
+copyright:           Copyright © 2019-2020 47 Degrees. <http://47deg.com>
+category:            Network
+build-type:          Simple
+cabal-version:       >=1.10
+extra-source-files:  README.md
+
+executable persistent-server
```

Comment:
I would be awesome if you could also create a client for this example. That way, we can tell people that they can run the server, and then make queries against it using the client.

Reply:
The client does not involve any DB, so it should be fairly easy to write.